rmgpy/cantherm/thermo.py

```diff
@@ -248,5 +247,10 @@ def plot(self, outputDirectory):
         pylab.ylabel('Gibbs free energy (kcal/mol)')
 
         fig.subplots_adjust(left=0.10, bottom=0.08, right=0.95, top=0.95, wspace=0.35, hspace=0.20)
-        pylab.savefig(os.path.join(outputDirectory, 'thermo.pdf'))
+
+        if not os.path.exists('plots'):
```

Comment:
Make `plots` in `outputDirectory`?

Reply:
'outputDirectory' becomes too packed if one has many plots, so I think it's better to have it in a subfolder